mautic/mautic
# Add the option to have Mautic update Contact Owner from Salesforce

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | mautic/documentation#128
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) |  #2893
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Adds the option to have the Salesforce integration update the Contact Owner based on the Salesforce Lead Owner.

#### Steps to test this PR:
1. Create a lead in salesforce
2. Ensure that a corresponding user with an _Identical_ email address exists in Mautic
3. Ensure that the Feature option in Salesforce plugin "Update Contact Owner" is enabled
4. Run `./app/console mautic:integration:fetchleads -i "Salesforce"`
5. Note that the new mautic Contact has their "Contact Owner" set to the correct Mautic User
